portal/src/graphql/portal/PasswordlessAuthenticatorScreen.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { FormattedMessage } from "@oursky/react-messageformat";
+import { Label, Text } from "@fluentui/react";
+import { useParams } from "react-router-dom";
+import cn from "classnames";
+import produce from "immer";
+import deepEqual from "deep-equal";
+
+import {
+  AppAndEmailSmsTemplatesConfigUpdater,
+  useUpdateAppAndEmailSmsTemplatesConfigMutation,
+} from "./mutations/updateAppAndEmailSmsTemplatesMutation";
+import { useAppAndEmailSmsTemplatesQuery } from "./query/appAndEmailSmsTemplatesQuery";
+import ShowError from "../../ShowError";
+import ShowLoading from "../../ShowLoading";
+import CodeEditor from "../../CodeEditor";
+import { clearEmptyObject } from "../../util/misc";
+import ButtonWithLoading from "../../ButtonWithLoading";
+import NavigationBlockerDialog from "../../NavigationBlockerDialog";
+import { PortalAPIAppConfig, PortalAPIEmailAndSmsTemplates } from "../../types";
+
+import styles from "./PasswordlessAuthenticatorScreen.module.scss";
+
+const EMAIL_HTML_TEMPLATE_NAME = "authenticate_secondary_oob_email.html";
+const EMAIL_MJML_TEMPLATE_NAME = "authenticate_secondary_oob_email.mjml";
```

Comment:
The MJML files should not be refererenced by portal

Reply:
Removed